x-pack/legacy/plugins/siem/public/store/timeline/selectors.ts

```diff
@@ -16,6 +16,9 @@ const selectTimelineById = (state: State): TimelineById => state.timeline.timeli
 
 const selectAutoSaveMsg = (state: State): AutoSavedWarningMsg => state.timeline.autoSavedWarningMsg;
 
+const selectcallOutUnauthorizedMsg = (state: State): boolean =>
```

Comment:
nit: `selectcallOutUnauthorizedMsg` could be changed to `selectCallOutUnauthorizedMsg` for consistency with other variables (i.e.  `showCallOutUnauthorizedMsg`)

Reply:
yes that's a mistake